src/elements/content-sidebar/activity-feed/comment/components/BaseCommentMenu.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import * as React from 'react';
+
+import { FormattedMessage } from 'react-intl';
+import TetherComponent from 'react-tether';
+
+import { ACTIVITY_TARGETS } from '../../../../common/interactionTargets';
+import { COMMENT_STATUS_OPEN, COMMENT_STATUS_RESOLVED } from '../../../../../constants';
+import { MenuItem } from '../../../../../components/menu';
+
+import Checkmark16 from '../../../../../icon/line/Checkmark16';
+import DeleteConfirmation from '../../common/delete-confirmation';
+import Media from '../../../../../components/media';
+import messages from '../messages';
+import Pencil16 from '../../../../../icon/line/Pencil16';
+import Trash16 from '../../../../../icon/line/Trash16';
+import X16 from '../../../../../icon/fill/X16';
+
+import type { FeedItemStatus } from '../../../../../common/types/feed';
+
+import './BaseCommentMenu.scss';
+
+export interface BaseCommentMenuProps {
+    canDelete: boolean;
+    canEdit: boolean;
+    canResolve: boolean;
+    handleDeleteCancel: () => void;
+    handleDeleteClick: () => void;
+    handleDeleteConfirm: () => void;
+    handleEditClick: () => void;
+    handleMenuClose: () => void;
+    handleStatusUpdate: (selectedStatus: FeedItemStatus) => void;
+    isConfirmingDelete: boolean;
+    isResolved: boolean;
+    onSelect: (isSelected: boolean) => void;
+}
+export const BaseCommentMenu = ({
+    canDelete,
+    canEdit,
+    canResolve,
+    handleDeleteCancel,
+    handleDeleteClick,
+    handleDeleteConfirm,
+    handleEditClick,
+    handleMenuClose,
+    handleStatusUpdate,
+    isConfirmingDelete,
+    isResolved,
+    onSelect,
+}: BaseCommentMenuProps) => {
+    return (
+        <div className="BaseCommentMenu">
```

Comment:
Is this wrapping div needed or can we add a class to the `Media.Menu` component?

Reply:
Ah, you're right. The lone override in this class doesn't need to apply to the `TetherComponent` here, nor to the `DropdownMenu` in `MediaMenu`, so it's okay to just let `MediaMenu` forward those classes to the `PlanButton` (as it does).